SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -1213,12 +1213,8 @@ void FSpatialGDKEditorToolbarModule::OnAutoStartLocalDeploymentChanged()
 		if (!UEditorEngine::TryStartSpatialDeployment.IsBound())
 		{
 			// Bind the TryStartSpatialDeployment delegate if autostart is enabled.
-			UEditorEngine::TryStartSpatialDeployment.BindLambda([this]
-			{
-				if (GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking())
-				{
-					VerifyAndStartDeployment();
-				}
+			UEditorEngine::TryStartSpatialDeployment.BindLambda([this](){
+				VerifyAndStartDeployment();
```

Comment:
This was changed in Mike's recent PR, I think - is the revert here intentional?

Reply:
I changed it because I noticed that inside Vertify the first thing it does is check is UsesSpatialNetworking, so maybe that was done to prevent having an error displayed some times? Let me know if I should put it back as it was @m-samiec

Reply:
Yes that was the reason - to prevent the error appearing when not running in spatial.

Reply:
Sorry about that, putting it back!